src/amo/components/Home/index.js

```diff
@@ -144,8 +144,10 @@ export class HomeBase extends React.Component {
               to={`/collections/mozilla/${collectionSlug}/`}
               className="Home-SubjectShelf-link"
             >
-              <Icon name={`Home-SubjectShelf-${collectionSlug}`} />
-              <span>{title}</span>
+              <Icon name={`Home-SubjectShelf-${collectionSlug}`}>
+                <span class="Home-SubjectShelf-link-text-vw">{title}</span>
```

Comment:
In React JSX you'll need to use DOM attribute names, not normal HTML attributes. In this case, `class=` should be `className=`.